QueryBuilder/Query.Aggregate.cs

```diff
@@ -82,6 +82,17 @@ public Query SelectCountDistinct(IEnumerable<string> columns, string alias = nul
         }
 
 
+        /**********************************************************************
+         ** AnyValue                                                         **
+         **********************************************************************/
+        // Note: The DISTINCT keyword can be specified for this function, but it
+        // does not have any effect.
```

Comment:
So? Same holds for MIN/MAX, and we still have implemented a `SelectMaxDistinct`/`SelectMinDistinct`. Why is this different?

I'm in favor of not having useless functions, but we should remove those as well then.

Reply:
rebased on top of #10